apps/taskmasters/flowlord/handler.go

```diff
@@ -55,8 +58,9 @@ func (tm *taskMaster) Info(w http.ResponseWriter, r *http.Request) {
 		}
 		wCache[key] = phases
 	}
-
-	for _, e := range tm.cron.Entries() {
+	entries := tm.cron.Entries()
+	for i := 0; i < len(entries); i++ {
+		e := entries[i]
```

Comment:
this seems a little odd to me, why would you change a range to an index that you build. Doesn't the range do the exact same thing?

Reply:
you can't see it in the review but its a pointer issue. I'm taking the pointer on that value and storing it. If I reuse the object every entry gets the same address which caused the problem with the times showing up the same. 
``` go 
ent := cEntry{
Next:     &e.Next,
Prev: &e.Prev,
```